spec-0007/index.md

```diff
@@ -24,11 +24,19 @@ We recommend:
 
 We suggest implementing these principles by:
 
-- deprecating the use of `numpy.random.seed` to control the random state,
+- deprecating user control of downstream library behavior with `numpy.random.seed`,
```

Comment:
I'm not sure what this sentence says now. Perhaps it should be "deprecating libraries dependence on `numpy.random.seed` for global random state"?

Reply:
I'm not recall whether this was motivated by one of Ralf's comments. I noted at some point that we didn't want to suggest that `np.random.seed` itself is decorated. Libraries can certainly continue to depend on `np.random.seed` for global random state (e.g. in tests), and users can continue to use `np.random.seed` for whatever they want (independent of the library). What is not allowed anymore is users calling `np.random.seed` to manipulate the random state used within library code.

Reply:
In that regard, the sentence was quite clear: stop using `np.random.seed` to control random state. I don't think it was saying "`np.random.seed` no longer controls random state".

But, to be explicit, we could say something along the lines of: "In NumPy, `numpy.random.seed` sets a global random seed. Downstream libraries should not make use of this global seed."

Alternatively, we keep the wording as is, since you have already added a sentence later that makes it explicit that `numpy.random.seed` is not deprecated?

Reply:
```suggestion
- deprecating the use of `numpy.random.seed` to control the random state,
```